modules/book/book.module

```diff
@@ -1109,13 +1109,6 @@ function template_preprocess_book_navigation(&$variables) {
       $variables['prev_title'] = check_plain($prev['title']);
     }
 
-    if ($book_link['plid'] && $parent = book_link_load($book_link['plid'])) {
-      $parent_href = url($parent['href']);
-      drupal_add_html_head_link(array('rel' => 'up', 'href' => $parent_href));
-      $variables['parent_url'] = $parent_href;
-      $variables['parent_title'] = check_plain($parent['title']);
-    }
-
```

Comment:
This whole section doesn't need to be removed as it also removes the visible "Up" link on the book page. In order to fix the validation issues, only line 1114 needs to be removed.